pkg/controller/controller.go

```diff
@@ -132,6 +132,11 @@ const (
 	annStorageProvisioner     = "volume.kubernetes.io/storage-provisioner"
 	annSelectedNode           = "volume.kubernetes.io/selected-node"
 
+	// Annotation for secret name and namespace will be added to the pv object
+	// and used at pvc deletion time.
+	AnnDeletionProvisionerSecretRefName      = "volume.kubernetes.io/provisioner-deletion-secret-name"
+	AnnDeletionProvisionerSecretRefNamespace = "volume.kubernetes.io/provisioner-deletion-secret-namespace"
```

Comment:
Why is it exported outside of the package?

Reply:
unexported